src/models/groupUsers.js

```diff
@@ -0,0 +1,21 @@
+module.exports = (sequelize, DataTypes) => {
+    const GroupUsers = sequelize.define('GroupUsers', {
```

Comment:
Since you're using a cross model reference using strings, consider creating an constants.js file at the src/models level and keeping all the model names there. This will help you to avoid typos in the model references.
E.g. inside constants.js
`const MODELS = { ..., GROUP_USERS: 'GroupUsers', ...};`

Reply:
resolved